lib/community/widgets/post_card_list.dart

```diff
@@ -359,15 +360,25 @@ class _PostCardListState extends State<PostCardList> with TickerProviderStateMix
 
   Future<void> dismissRead() async {
     if (widget.postViews != null) {
-      setState(() {
-        showRead = false;
-      });
-      await Future.delayed(const Duration(milliseconds: 400));
+      if (!(widget.postViews!.length > 15)) {
```

Comment:
Just a quick question - why are there two calls to `widget.onScrollEndReached();`? One here and one on line 380

Reply:
Ahh okay - I think we can make the second call a bit more robust in this case. I actually had to do this to resolve a issue where posts would not load on tall screens.

In my case, I had to do it when the state was first initiated, but for your case, we can make do without `WidgetsBinding.instance.addPostFrameCallback((_) {}`

```dart
// Check to see if the initial load did not load enough items to allow for scrolling to occur and fetches more items
WidgetsBinding.instance.addPostFrameCallback((_) {
  bool isScrollable = _scrollController.position.maxScrollExtent > _scrollController.position.viewportDimension;

  if (context.read<CommunityBloc>().state.hasReachedEnd == false && isScrollable == false) {
    widget.onScrollEndReached();
  }
});
```

Reply:
Done